src/test/java/org/llorllale/cactoos/matchers/MatcherEnvelopeTest.java

```diff
@@ -104,8 +104,9 @@ public void decoratesDescribeMismatchSafely() {
                     "must delegate describeMismatchSafely to encapsulated matcher",
                     MatcherEnvelopeTest.TEST_INTEGER,
                     new MatcherEnvelopeChild<>(
-                        (item) -> false,
-                        (description) -> { },
+                        item -> false,
+                        description -> {
+                        },
```

Comment:
@andreoss can't this stay on one line? maybe by removing the space between `{` and `}`?